packages/conf-perl-packages/conf-perl-packages.1/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "chetsky@gmail.com"
+homepage: "https://www.perl.org/"
+bug-reports: "chesky@gmail.com"
+license: "GPL-1.0-or-later"
+authors: "Larry Wall et. al."
+depends: [
+  "conf-perl"
+]
+build: [
+  ["perl" "--version"]
+  ["perl" "-MIPC::System::Simple" "-e" "1"]
+  ["perl" "-MString::ShellQuote" "-e" "1"]
+]
+depexts: [
+  [
+    "libstring-shellquote-perl"
+    "libipc-system-simple-perl"
+  ] {os-family = "debian"}
+  [
+    "perl-string-shellquote"
+    "perl-ipc-system-simple"
+  ] {os-distribution = "alpine"}
+  [
+    "perl-String-ShellQuote"
+    "perl-IPC-System-Simple"
+  ] {os-distribution = "centos"}
```

Comment:
```suggestion
  ] {os-distribution = "centos" | os-distribution = "ol"}
```
I've tested it using the `ocaml/opam:oraclelinux` docker image and both centos and oraclelinux have these two packages

Reply:
Ah nevermind, it works with oraclelinux 8 but not oraclelinux 7. I'll ignore the error on oraclelinux 7

Reply:
How odd.  OK, I'll enable that and either it goes thru, or we'll have a testcase.  How odd.  I am -very- sure that both centos and ol had CI failures with these packages.

Reply:
Well, I also saw errors on centos-7.  Since I've already made the changes, I'll let it go thru CI (which will be -fast-, b/c early failures) and we'll see).